src/applications/static-pages/static-pages-entry.js

```diff
@@ -101,6 +101,9 @@ import create1010DAccess from './ivc-champva/10-10D/entry';
 import create107959CAccess from './ivc-champva/10-7959c/entry';
 import create107959F1Access from './ivc-champva/10-7959f-1/entry';
 
+import './mhv-signin-cta/sass/mhv-signin-cta.scss';
```

Comment:
Importing this SASS here allow us to unit test the other files as @imports breaks the unit test framework.